src/logistration/RegistrationPage.jsx

```diff
@@ -153,7 +153,7 @@ class RegistrationPage extends React.Component {
 
   renderCountryList() {
     const locale = getLocale();
```

Comment:
This code is removed in my PR. We dont need this as the backend is giving us country list. https://github.com/edx/frontend-app-logistration/pull/31/files#diff-e871e903f58eef8e0d9182a0de24715ead7eb4cb866505c91e1ac1f329811793L154